src/internal/easee/auth.go

```diff
@@ -157,6 +162,25 @@ func (a *authenticator) handleFailedRefreshToken(err error) error {
 	}
 }
 
+func (a *authenticator) triggerAppLogout() error {
```

Comment:
I'd rather call `authenticator.Logout` directly instead of sending an MQTT message. You can move lifecycle manipulation from `app.Logout`, as it will be needed in this case.

Reply:
I preferred sending a message as it's a proper way to invoke a full logout sequence, not just a lifecycle.
It's rather independent of the logout implementation.

Reply:
Following our conversation added a comment explaining why we use this approach right now.